cmd/golangorg/server.go

```diff
@@ -75,13 +74,13 @@ func main() {
 	// so that updates to those files appear on the local dev instance without restarting.
 	// On App Engine, leave contentDir empty, so we use the embedded copy,
 	// which is much faster to access than the simulated file system.
-	if *contentDir == "" && !runningOnAppEngine {
-		repoRoot := "../.."
-		if _, err := os.Stat("_content"); err == nil {
-			repoRoot = "."
-		}
-		*contentDir = filepath.Join(repoRoot, "_content")
-	}
+	// if *contentDir == "" && !runningOnAppEngine {
```

Comment:
Could you please leave this code as is given that we shall detect if a localrun? Or rather how come you commented out this code?

Reply:
The commenting out is expected to be temporary. This was when Jamal was helping debug why the container was not working. After this, and after changing the dockerfile listen address to 0.0.0.0:port did it work. But using localrun is a better choice in retrospect.